### PR TITLE
cleanup from previous PR

### PR DESCRIPTION
**Pull request by cworsnop-figure**

### CODE REVIEW

File: .github/workflows/release-build-publish.yml [hunk 1]
```diff
@@ -1,25 +1,26 @@
...
-      OS_GRPC_URL: grpc://object-store-v2.p8e:80
+      OS_GRPC_URL: grpc://object-store-v2.core:80
       PROVENANCE_GRPC_URL: grpc://192.168.1.244:9090
-      TEST_ENCRYPTION_PRIVATE_KEY: ${{ secrets.TEST_PRIVATE_KEY }}
-      TEST_SIGNING_PRIVATE_KEY: ${{ secrets.TEST_PRIVATE_KEY }}
-      PROD_ENCRYPTION_PRIVATE_KEY: ${{ secrets.PROD_PRIVATE_KEY }}
-      PROD_SIGNING_PRIVATE_KEY: ${{ secrets.PROD_PRIVATE_KEY }}
-      TEST_CHAIN_ID: pio-testnet-1
-      PROD_CHAIN_ID: pio-mainnet-1
+      ENCRYPTION_PRIVATE_KEY: IS_TEST && ${{ secrets.TEST_PRIVATE_KEY }} || ${{ secrets.PROD_PRIVATE_KEY }}
```
Comment (rpatel-figure) [inline .github/workflows/release-build-publish.yml:22]: I don't see `IS_TEST` defined anywhere? 😅

File: .github/workflows/release-build-publish.yml [hunk 2]
```diff
@@ -46,9 +47,5 @@ jobs:
...
         run: ./gradlew p8eClean p8eCheck --info
 
-        # in order to bootstrap in github actions - we must have an object-store and query node that we can hit
       - name: Bootstrap Test
-        run: CHAIN_ID=$TEST_CHAIN_ID ENCRYPTION_PRIVATE_KEY=$TEST_ENCRYPTION_PRIVATE_KEY SIGNING_PRIVATE_KEY=$TEST_SIGNING_PRIVATE_KEY ./gradlew p8eBootstrap --info
-
-#      - name: Bootstrap Prod
-#        run: CHAIN_ID=$PROD_CHAIN_ID ENCRYPTION_PRIVATE_KEY=$PROD_ENCRYPTION_PRIVATE_KEY SIGNING_PRIVATE_KEY=$PROD_SIGNING_PRIVATE_KEY ./gradlew p8eBootstrap --info
\ No newline at end of file
+        run: CHAIN_ID=$CHAIN_ID ENCRYPTION_PRIVATE_KEY=$ENCRYPTION_PRIVATE_KEY SIGNING_PRIVATE_KEY=$ENCRYPTION_PRIVATE_KEY IS_TEST=${{ github.event.inputs.publishing_environment == 'testnet' }} ./gradlew p8eBootstrap --info
```
Comment (rpatel-figure) [inline .github/workflows/release-build-publish.yml:51]: If you do define `IS_TEST`, may want to update this to
```suggestion
        run: CHAIN_ID=$CHAIN_ID ENCRYPTION_PRIVATE_KEY=$ENCRYPTION_PRIVATE_KEY SIGNING_PRIVATE_KEY=$ENCRYPTION_PRIVATE_KEY IS_TEST=$IS_TEST ./gradlew p8eBootstrap --info
```

File: .github/workflows/release-build-publish.yml [hunk 3]
```diff
@@ -1,25 +1,26 @@
...
+  workflow_dispatch:
+    inputs:
+      publishing_environment:
+        description: 'environment the contracts should be bootstrapped to'
+        required: true
+        default: 'testnet'
+        type: boolean
+        options:
+          - 'testnet'
+          - 'mainnet'
```
Comment (rpatel-figure) [inline .github/workflows/release-build-publish.yml:13]: Type is boolean but value options and default value are strings?

File: .github/workflows/build.yml [hunk 4]
```diff
@@ -30,4 +30,7 @@ jobs:
         run: ./gradlew clean build --refresh-dependencies -Pversion=$VERSION
         env:
           GITHUB_ACTOR: ${{ github.actor }}
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
\ No newline at end of file
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
Comment (rpatel-figure) [inline .github/workflows/build.yml]: I wonder if these are necessary, and if so, what they are used for

**Review by rpatel-figure — COMMENTED** [review]
Looks like you used
```yaml
${{ github.event.inputs.publishing_environment == 'testnet' }}
```
in most places but in other places, used `IS_TEST` without defining it as a boolean anywhere?

File: .github/workflows/build.yml (hunk 4 above)
Comment (cworsnop-figure) [reply .github/workflows/build.yml]: great question xD Let me remove them and see what happens :P

File: .github/workflows/release-build-publish.yml (hunk 1 above)
Comment (cworsnop-figure) [reply .github/workflows/release-build-publish.yml]: :awkward: fixed

**Pull request merged**